main.py

```diff
@@ -116,6 +116,64 @@ def add_mail(self, mail):
     def add_birthday(self, bd):
         self.birthday = bd
         print(f"{self.name} was born {bd}.")
+        
+    def change_adress(self, new_adres, old_adres):
+        adress_list = []
+        for a.value in self.adresses:
+            adress_list.append(str(a.value))
+        if old_adres in adress_list:
+            old_adress = Adress(old_adres)
+            self.adresses.remove(old_adress)
+            new_adress = Adress(new_adres)
+            self.adresses.append(new_adress)
+        else:
+            print(f"This adress {old_adres} is not in the list")
+            
+    def change_phone(self, new_num, old_num):
+        phone_list = []
+        for p.value in self.phone:
+            phone_list.append(str(p.value))
+        if old_num in phone_list:
+            old_phone = Phone(old_num)
+            self.phones.remove(old_phone)
+            new_phone = Phone(new_num)
```

Comment:
Please add Phone(new_num) for correctnes(smth like: if Phone.value = None) otherwise it will be reated and rewritten anyway